kata-deploy/scripts/kata-deploy.sh

```diff
@@ -57,10 +57,13 @@ function configure_crio() {
 	# backup the CRIO.conf only if a backup doesn't already exist (don't override original)
 	cp -n "$crio_conf_file" "$crio_conf_file_backup"
 
-	cat <<EOT | tee -a "$crio_conf_file"
+	grep -Fq '[crio.runtime.runtimes.kata-qemu]' $crio_conf_file \
```

Comment:
This PR avoids overriding the configuration if it already exists. Not sure, what the best strategy should be.

Reply:
I wonder if an error being returned would be wise here? I’m not sure best way to handle, but silently overwriting and silently ignoring aren’t great.

Reply:
Overwrite just like network lifecycle. Document the behavior?

Reply:
We backup the file 2 lines above, so... ? :)

Reply:
@amshinde can you follow @krsna1729's recommendation?  Since we save the file already, I think overwriting does make sense, put perhaps throw a warning